pkg/util/mount/mount_linux.go

```diff
@@ -198,7 +198,19 @@ func (mounter *Mounter) Unmount(target string) error {
 	command := exec.Command("umount", target)
 	output, err := command.CombinedOutput()
 	if err != nil {
-		return fmt.Errorf("Unmount failed: %v\nUnmounting arguments: %s\nOutput: %s\n", err, target, string(output))
+		glog.Errorf("Unmount failed: %v\nUnmounting arguments: %s\nOutput: %s\n", err, target, string(output))
+		return LazyUnmount(target)
+	}
+	return nil
+}
+
+// Unmount unmounts the target with lazy option.
+func LazyUnmount(target string) error {
```

Comment:
How is this different from simply retrying umount if it failed because device is busy? 

As I understand it, lazy umount removes the file system from hierarchy right away but other references are cleaned up later. That could be problematic because, for attachable volume types - we would perform detach right after umount is successful but success in case of lazy umount gives wrong impression and can result in corrupt file systems.

Reply:
I want to delete a pod,but umount volume always failed,the pod stuck in Terminating state.
I hope the deleting pod operation success in short time.
in case of nfs volume，does LazyUmount work？

Reply:
I think we should work on finding out, why is normal umount not completing in right amount of time rather than using lazy umount.  cc @rootfs

Reply:
+1, understanding what causes the problem is better than a workaround, especially the workaround has the potential to mask the real trouble.

Reply:
ok,thanks